api_v3/lib/types/KalturaPlaybackContext.php

```diff
@@ -16,28 +16,22 @@ class KalturaPlaybackContextResult extends KalturaObject{
 	public $flavorAssets;
 
 	/**
-	 * Array of messages as received from the rules that invalidated
-	 * @var KalturaStringArray
-	 */
-	public $messages;
-
-	/**
 	 * Array of actions as received from the rules that invalidated
 	 * @var KalturaRuleActionArray
 	 */
 	public $actions;
 
 	/**
 	 * Array of actions as received from the rules that invalidated
-	 * @var KalturaPlaybackRestrictionsArray
+	 * @var KalturaAccessControlMessageArray
```

Comment:
I don't understand why the code should be accompanied with the message, the actions array should be enough.